src/App.js

```diff
@@ -64,6 +64,7 @@ const App = () => {
             <Redirect from='/tag-creator' to='/tag-generator' />
             <Redirect from='/tagcreator' to='/tag-generator' />
             <Redirect from='/search' to='/projects' />
+            <Redirect from='/support' to='/radicalcollaboration' />
             <Redirect from='/faq' to='/radicalcollaboration/faq' />
             <Redirect from='/radicalcollaboration' to='/radicalcollaboration/faq' component={Faq} />
```

Comment:
Remove this Redirect since we want /radicalcollaboration to go to the SupportUs page